Sources/MongoSwift/Operations/FindOperation.swift

```diff
@@ -32,6 +32,9 @@ public enum CursorType {
 
 /// Options to use when executing a `find` command on a `MongoCollection`.
 public struct FindOptions: Codable {
+    /// Enables writing to temporary files on the server.
```

Comment:
I swear I posted this comment in my first review but I guess not.... I think the second sentence from the CRUD spec about this option would also be useful to tack on here as it clarifies the temp files are just for the duration of the find.
>  When set to true, the server can write temporary data to disk while executing the find operation.